### PR TITLE
[Task] Update Documentation for GEE and custom attributes

### PR DESCRIPTION
**Pull request by lukmzig**
## Changes in this pull request
Pat of https://github.com/pimcore/studio-backend-bundle/issues/350

### CODE REVIEW

File: doc/05_Additional_Custom_Attributes.md [hunk 1]
```diff
@@ -80,6 +95,14 @@ final class AssetEvent extends AbstractPreResponseEvent
 }
 ```
 
+### List of custom attributes
+
+- `icon` - The custom icon that should be displayed in the tree
+- `tooltip`- The custom HTML tooltip to be displayed in the tree
+- `additionalIcons` - Array of additional icons that should be displayed in the tree
+- `key` - The key that should be displayed in the tree
+- `additionalCssClasses` - Additional CSS classes that should be added to the tree element
```
Comment (martineiber) [inline doc/05_Additional_Custom_Attributes.md]:
```suggestion
- `icon` - The custom icon that should be displayed in the tree.
- `tooltip`- The custom HTML tooltip to be displayed in the tree.
- `additionalIcons` - Array of additional icons that should be displayed in the tree.
- `key` - The key that should be displayed in the tree.
- `additionalCssClasses` - Additional CSS classes that should be added to the tree element.
```

**Review by martineiber — COMMENTED** [review]
Just a small change but i think we should add a dot here in the end?

File: doc/05_Additional_Custom_Attributes.md (hunk 1 above)
Comment (mattamon) [reply doc/05_Additional_Custom_Attributes.md]: 🐧 supports the dots! +1

Comment (lukmzig) [reply doc/05_Additional_Custom_Attributes.md]: I agree, thanks! :)

**Pull request merged**